recipe/meta.yaml

```diff
@@ -172,6 +180,9 @@ test:
     # 2024/02 These tests assert warnings and in PyTorch 2.1.2 the number of warnings increased
     # causing them to fail
     {% set tests_to_skip = tests_to_skip + " or test_pretrained_pos or test_equivalent_behavior_weights" %}
+    # use pip list to manually check that the version is correctly reported
+    - pip list
```

Comment:
Rather than relying on manual verification, we could combine this into more concrete tests, e.g. add `| grep torchvision | grep {{ version }}` or something like that. C.f. also https://github.com/conda-forge/cvxpy-feedstock/commit/1c70e7af0f38a8ac334f797299696f19c4d6d885

Reply:
thanks